cves/kernel/CVE-2022-1671.yml

```diff
@@ -131,10 +130,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: "No unit tests are present in the project."
+  fix: true
```

Comment:
Same comment as above on unit tests.. it doesn't look like the fix commit adds/updates any unit tests, can you clarify why you think this is 'true'?